metabase/metabase
# Clean up notebook `SortStep` component

Epic #28034

Cleans up and adds tests to notebook editor's `SortStep` component. Mostly a preparation to convert it to MLv2

### How to verify

1. Open `/question/:id/notebook`
2. Click the "Sort" button to add a sorting step
3. Play around with adding/updating/removing sorts and ensure the behavior matches `master`


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/29721)
